### PR TITLE
Megafauna are no longer chronically afraid of chasms

### PR DESCRIPTION
**Pull request by Qwertytoforty**
## What Does This PR Do

Megafauna now path across chasms to murder people that are on the other side.
Megafauna will also fuck up structures around them more often.

## Why It's Good For The Game

Being able to cheese megafauna with chasms bad.
Megafauna causing more destruction good?

## Testing

Taught megafauna to not fear the abyss, confirmed could path across

## Changelog
:cl:
fix: Megafauna are no longer chronically afraid of chasms
/:cl:

### CODE REVIEW

File: code/game/turfs/simulated/floor/chasm.dm [hunk 1]
```diff
@@ -44,6 +45,14 @@
 	if(!drop_stuff())
 		STOP_PROCESSING(SSprocessing, src)
 
+/turf/simulated/floor/chasm/CanPathfindPass(obj/item/card/id/ID, to_dir, caller, no_id = FALSE)
+	if(isliving(caller))
+		var/mob/living/L = caller
+		if(L.flying || ismegafauna(caller))
+			return TRUE
+		return FALSE
+	return TRUE
```
Comment (Contrabang) [inline code/game/turfs/simulated/floor/chasm.dm:54]:
```suggestion
	if(!isliving(caller))
		return TRUE
	var/mob/living/L = caller
	return (L.flying || ismegafauna(caller)
```

**Review by Contrabang — APPROVED** [review]
i hate it too, but LGTM

**Comment by Qwertytoforty** [comment]
> code looks fine, just needs resolving with CI. I think something went wrong with tab/space spacing

fixed, you forgot a )

**Comment by Contrabang** [comment]
> > code looks fine, just needs resolving with CI. I think something went wrong with tab/space spacing
> 
> fixed, you forgot a )

makes sense, LGTM

**Pull request merged**